ext/standard/file.c

```diff
@@ -917,6 +918,34 @@ PHPAPI PHP_FUNCTION(fclose)
 }
 /* }}} */
 
+/* {{{ proto bool freopen(string filename, string mode, resource fp)
+   Reopen an open file pointer */
+PHPAPI PHP_FUNCTION(freopen)
+{
+	char *filename, *mode;
+	int filename_len, mode_len;
+	zval *arg1;
+	php_stream *stream;
+
+	if (zend_parse_parameters(ZEND_NUM_ARGS() TSRMLS_CC, "psr", &filename, &filename_len, &mode, &mode_len, &arg1) == FAILURE) {
```

Comment:
Oh, somehow I've missed the moment those were removed.
I've nuked them all now.